linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/oval/shared.xml

```diff
@@ -14,6 +14,8 @@
   <ind:textfilecontent54_object id="object_no_empty_passwords" version="1">
 {{% if product in ['sle12', 'sle15'] %}}
     <ind:filepath operation="pattern match">^/etc/pam.d/.*$</ind:filepath>
+{{% elif product in ['ubuntu1804', 'ubuntu2004'] %}}
```

Comment:
please use instead: `if 'ubuntu' in product`

Reply:
Updated, thanks for review